components/centraldashboard/license_info.csv

```diff
@@ -0,0 +1,1287 @@
+content-type@1.0.4,https://github.com/jshttp/content-type/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/jshttp/content-type/master/LICENSE
+convert-source-map@1.6.0,https://github.com/thlorenz/convert-source-map/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/thlorenz/convert-source-map/master/LICENSE
+cookie-signature@1.0.6,https://github.com/visionmedia/node-cookie-signature/tree/master/Readme.md,MIT,https://raw.githubusercontent.com/visionmedia/node-cookie-signature/master/Readme.md
+cookie@0.3.1,https://github.com/jshttp/cookie/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/jshttp/cookie/master/LICENSE
+cookie@0.4.0,https://github.com/jshttp/cookie/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/jshttp/cookie/master/LICENSE
+copy-concurrently@1.0.5,https://github.com/npm/copy-concurrently/tree/latest/LICENSE,ISC,https://raw.githubusercontent.com/npm/copy-concurrently/latest/LICENSE
+copy-descriptor@0.1.1,https://github.com/jonschlinkert/copy-descriptor/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/jonschlinkert/copy-descriptor/master/LICENSE
+copy-webpack-plugin@5.0.4,https://github.com/webpack-contrib/copy-webpack-plugin/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/webpack-contrib/copy-webpack-plugin/master/LICENSE
+core-js-compat@3.2.1,https://github.com/zloirock/core-js/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/zloirock/core-js/master/LICENSE
+core-js@2.6.9,https://github.com/zloirock/core-js/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/zloirock/core-js/master/LICENSE
+core-js@3.2.1,https://github.com/zloirock/core-js/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/zloirock/core-js/master/LICENSE
+core-util-is@1.0.2,https://github.com/isaacs/core-util-is/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/isaacs/core-util-is/master/LICENSE
+cp-file@6.2.0,https://github.com/sindresorhus/cp-file/tree/master/license,MIT,https://raw.githubusercontent.com/sindresorhus/cp-file/master/license
+create-ecdh@4.0.3,https://github.com/crypto-browserify/createECDH/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/crypto-browserify/createECDH/master/LICENSE
+create-error-class@3.0.2,https://github.com/floatdrop/create-error-class/tree/master/license,MIT,https://raw.githubusercontent.com/floatdrop/create-error-class/master/license
+create-hash@1.2.0,https://github.com/crypto-browserify/createHash/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/crypto-browserify/createHash/master/LICENSE
+create-hmac@1.1.7,https://github.com/crypto-browserify/createHmac/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/crypto-browserify/createHmac/master/LICENSE
+cross-env@5.2.1,https://github.com/kentcdodds/cross-env/tree/master/LICENSE,MIT,https://raw.githubusercontent.com/kentcdodds/cross-env/master/LICENS
```

Comment:
Unlicense? If that is the case, can we get rid of this dep?

Reply:
It is a valid license: (similar to MIT). Let's check on it?

https://github.com/dchest/tweetnacl-js/tree/master/LICENSE